Update overview.md

BenShy opened the pull request:
Updating URL to the Azure Cost Management Overview and Demo to https://azure.microsoft.com/en-us/resources/videos/azure-cost-management-overview-and-demo

azurecla commented:
Hi __@olgamatoula__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>
        This seems like a small (but important) contribution, so no contribution license agreement is required at this point. Real humans will now evaluate your PR.
    </span>

TTYL, AZPRBOT;

tysonn commented:
@olgamatoula - Closing per issue #6002

BenShy committed:
Update overview.md

Updating URL to the Azure Cost Management Overview and Demo to https://azure.microsoft.com/en-us/resources/videos/azure-cost-management-overview-and-demo
